contracts/strategies/gmxFrf/libraries/LiquidationLogic.sol

```diff
@@ -158,18 +160,20 @@ library LiquidationLogic {
             );
         }
 
-        // Call the liquidation callback handler.
-        return (
-            rebalanceAmount,
-            SwapCallbackLogic.handleSwapCallback(
+        {
```

Comment:
maybe put an inline comment above? Don't feel very strongly

Reply:
Sure

Reply:
Well actuallly
I dont think it really matters, its pretty clear imo